bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -490,16 +491,23 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
 
     std::shared_ptr<Recorder> timeInActiveView =
         std::make_shared<Recorder>(1, MAX_VALUE_SECONDS, 3, concord::diagnostics::Unit::SECONDS);
+
+    std::shared_ptr<Recorder> timeInStateTransfer =
```

Comment:
I'm not sure 5 minutes is a long enough duration here. Maybe make it a few hours for max? I wouldn't change the constant above though, as making the duration longer makes the histograms larger and for most histograms we don't need hours.

Reply:
MAX_VALUE_SECONDS is set to 5 years, Am I missing something?

Reply:
Whoops. I misread. Sorry.